test/celeritas/em/CoulombScattering.test.cc

```diff
@@ -368,96 +380,101 @@ TEST_F(CoulombScatteringTest, wokvi_transport_xs)
 
 TEST_F(CoulombScatteringTest, simple_scattering)
 {
-    int const num_samples = 4;
+    int const num_samples = 8;
 
     auto const material = this->material_track().make_material_view();
     IsotopeView const isotope
         = material.make_element_view(elcomp_id_).make_isotope_view(isocomp_id_);
     auto cutoffs = this->cutoff_params()->get(mat_id_);
 
-    RandomEngine& rng_engine = this->rng();
+    auto& rng_engine = this->rng();
+
+    // Create one params for each form factor
+    std::vector<SPWentzel> all_wentzel;
+    std::vector<std::string> ff_str;
+    for (auto ff : range(NuclearFormFactorType::size_))
+    {
+        all_wentzel.push_back(this->make_wentzel_params(ff));
+        ff_str.push_back(to_cstring(ff));
+    }
 
     std::vector<real_type> cos_theta;
-    std::vector<real_type> delta_energy;
+    std::vector<real_type> eloss_frac;
 
-    std::vector<real_type> energies{0.2, 1, 10, 100, 1000, 100000};
-    for (auto energy : energies)
+    for (auto particle : {pdg::electron(), pdg::positron()})
     {
-        this->set_inc_particle(pdg::electron(), MevEnergy{energy});
-        CoulombScatteringInteractor interact(model_->host_ref(),
-                                             wentzel_->host_ref(),
-                                             this->particle_track(),
-                                             this->direction(),
-                                             material,
-                                             isotope,
-                                             el_id_,
-                                             cutoffs);
-
-        for ([[maybe_unused]] int i : range(num_samples))
+        for (real_type energy : {0.05, 0.1, 0.5, 1.0, 10.0})
```

Comment:
Yeah good point, and probably below 50 keV too. I chose these since they were the energies used in the paper showing Mott factors, but of course we're going to be using the scattering kernel outside of that.